Workbooks/WorkbooksMetadata.json

```diff
@@ -905,5 +905,18 @@
     "templateRelativePath": "CyberArkEPV.json",
     "subtitle": "",
     "provider": "CyberArk"
+  },
+  {
+  "workbookKey": "EntityBehaviorAnalyticsWorkbook",
+  "logoFileName": "Microsoft_logo.svg",
+  "description": "Hunt for compromised users and insider threats by leveraging entity behavior analytics, gain insights on user behavior and anomalies ",
```

Comment:
change the description to the following:

Identify compromised users and insider threats using entity behavior analytics. Gain insights into anomalous user behavior from baselines learned from behavior patterns.